### ISSUE TITLE
`ugbt list gox` anomaly

### ISSUE THREAD

**Issue by georgettica** [kortschak/ugbt#7]
It seems it won't note I'm on the latest version as gox doesn't have the time each release is cut

**Comment by kortschak** [comment]
Can you provide details please?

**Comment by kortschak** [comment]
It seems that the last released version of gox is v1.0.1 in 2019.

```
$ go install github.com/mitchellh/gox@v0.3.0                
go: downloading github.com/mitchellh/gox v0.3.0
go: finding module for package github.com/mitchellh/iochan
go: found github.com/mitchellh/iochan in github.com/mitchellh/iochan v1.0.0
$ ugbt list gox                             
v1.0.1  10 Apr 2019 15:02
v1.0.0  25 Oct 2018 21:53
v0.4.0  25 Jul 2016 18:46
$ ugbt install gox v1.0.1      
$ ugbt list gox          
no new version
```
